### DISCUSSION TITLE
number( ) not expect a 'Complex' type

### DISCUSSION THREAD

**Discussion by Yuxin-Lou** [josdejong/mathjs#3155] [Q&A]
I can't run the example code in [document](https://mathjs.org/docs/expressions/syntax.html)
![image](https://github.com/josdejong/mathjs/assets/117000057/c5a24817-f274-49b5-8a05-42497a402a6c)

number( ) seems not expect a 'Complex' type.
![image](https://github.com/josdejong/mathjs/assets/117000057/577c9183-d807-4b8b-b639-762b533b676a)

**Comment by josdejong** [comment]
Thanks for reporting. This is an outdated section in the docs. I've removed it via https://github.com/josdejong/mathjs/commit/77d13c659fbb88686927ee730693404446eceee1.

To convert a complex number into a normal number, you can use functions `re` and `im`.
